src/Qir/Runtime/lib/QIR/QubitManagerRestrictedReuse.cpp

```diff
@@ -0,0 +1,468 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+// This code compiles, but was never executed, be careful!
```

Comment:
Now that you have a unit test for this, perhaps the comment can be updated to say this is prototype code that the runtime does not yet use?